### PR TITLE
DAT-3368 add create table sql schema

### PR DESCRIPTION
**Pull request by idradm**
https://wikia-inc.atlassian.net/browse/DAT-3368

### CODE REVIEW

File: service/template-classification-storage/sql/template_classification.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    page_id INT NOT NULL,
+    type VARCHAR(15) NOT NULL,
+    modified TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    PRIMARY KEY(wiki_id, page_id)
+);
+
+CREATE TABLE raw_types (
+    wiki_id INT NOT NULL,
+    page_id INT NOT NULL,
+    provider VARCHAR(256) NOT NULL,
```
Comment (sqreek) [inline service/template-classification-storage/sql/template_classification.sql]: Can we consider to use 100 chars or less?

Comment (SebastianMarzjan) [inline service/template-classification-storage/sql/template_classification.sql]: Also, please remember about describing the purpose of fields

Comment (idradm) [inline service/template-classification-storage/sql/template_classification.sql]: It's described: https://one.wikia-inc.com/wiki/Template_Classification_Storage

Comment (idradm) [inline service/template-classification-storage/sql/template_classification.sql]: Should I limit provider and origin strings?

Comment (SebastianMarzjan) [inline service/template-classification-storage/sql/template_classification.sql]: It makes sense to me to do so - I find it challenging to figure out a reason to have a provider name longer than 32 bytes. It'd be useful to add currently existing / predicted types to the documentation too.

**Comment by sqreek** [comment]
:+1:

**Pull request merged**